internal/model/model.go

```diff
@@ -42,12 +45,16 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 		}
 	}
-
 	return m, nil
 }
 
 func (m Model) View() string {
-	r, _ := glamour.NewTermRenderer(glamour.WithStylesFromJSONBytes(styles.Theme))
+	style := glamour.WithStyles(m.Theme)
+	if m.CustomTheme != nil {
+		style = m.CustomTheme
+	}
+
+	r, _ := glamour.NewTermRenderer(style)
```

Comment:
I think we should have the `glamour.TermRenderer` on the model, this way we don't need the `Theme` or `CustomTheme` on the model.

Reply:
Smart. Keeps it simple.

Reply:
Got this working by merging all theme selection under `styles.SelectTheme()` and setting the model to have a RenderOption. If I do it with the full renderer it's a bit harder to test equality between the two, but RenderOption seemed to yield the same end result.